NorthstarDLL/util/timer.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <chrono>
+
+//-----------------------------------------------------------------------------
+// Purpose: Timer class to help keep track of time
+//-----------------------------------------------------------------------------
+class CTimer
+{
+  public:
+	void Start(void);
+	long long GetDuration(void) const;
+
+  private:
+	std::chrono::time_point<std::chrono::steady_clock> m_Clock;
+};
+
+//-----------------------------------------------------------------------------
+// Purpose:
+//-----------------------------------------------------------------------------
```

Comment:
mfw

Reply:
![image](https://github.com/R2Northstar/NorthstarLauncher/assets/64418963/2098e008-587a-4d65-b9eb-5ce90d0eae5c)

This is on main, unless you have a better idea and want to format all of this we stick with this :)

Reply:
at least write what is does exactly